flint-cli/pom.xml

```diff
@@ -51,7 +51,7 @@
                             </descriptorRefs>
                             <archive>
                                 <manifest>
-                                    <mainClass>uk.bl.dpt.qa.flint.PolicyPropertiesCreatorApp</mainClass>
+                                    <mainClass>PolicyPropertiesCreatorApp</mainClass>
                                 </manifest>
```

Comment:
Does PolicyPropertiesCreatorApp and CoreApp need the package qualifiers

Reply:
I don't know. My IDE did this by itself, but I'll revert it just in case.